### DISCUSSION TITLE
> Ideas discussion page!

### DISCUSSION THREAD

**Discussion by nikhil25803** [Clueless-Community/collegeAPI#18] [Ideas]
## Hi👋 
  We’re using Discussions as a place to connect with other members of our community and share your idea to enhance this project. 
We hope that you:
  * Share your idea.
  * Explain your goal and how you will accomplish them
  * Engage with other community members.
  * Welcome others and are open-minded. 
  
**Remember that this is a community we build together 💪**

  To get started, comment below with an introduction of yourself 🫂

**Comment by nikhil25803** [comment]
@Rajdip019  | @Roshaen  | @Debajyoti14 
Inviting you guys to discuss the UI inspiration for this project.
As of now, I think that we can take inspiration from [this](https://animechan.vercel.app/) website.

**Reply by lostgirljourney** [reply]
Looks good. We can look up dribbble and Behance for inspiration as well.

**Reply by nikhil25803** [reply]
Yes, feel free to drop the link here if you find a relevant one.

**Reply by lostgirljourney** [reply]
Hey, I guess taking inspiration from the website you mentioned is enough. We can start working on this.

**Reply by nikhil25803** [reply]
Yes, @lostgirljourney | Go ahead

**Comment by KshitizPandya** [comment]
> ## Hi👋
> We’re using Discussions as a place to connect with other members of our community and share your idea to enhance this project. We hope that you:
> 
> * Share your idea.
> * Explain your goal and how you will accomplish them
> * Engage with other community members.
> * Welcome others and are open-minded.
> 
> **Remember that this is a community we build together 💪**
> 
> To get started, comment below with an introduction of yourself 🫂


Hi @nikhil25803 👋 , I came to know about your project + repository from Winter of Code 2023 from IIIT Kalyani.
Your project was floated over there giving the basic idea definition of the project, given in your documentation.
I have recently worked on a similar, but way simpler project for extracting the details of NIRF, NAAC, and NBA rankings of different categories and colleges and publishing on a web-app.
As I have a basic knowledge of what this project is suggesting, I wanted to know more clearly what will we be working on if my proposal will be accepted. 

Looking forward to your reply.
